pkg/flowkit/services/events_test.go

```diff
@@ -34,74 +36,81 @@ func TestEvents(t *testing.T) {
 		t.Parallel()
 
 		_, s, gw := setup()
-		_, err := s.Events.Get("flow.CreateAccount", "0", "1")
+		_, err := s.Events.Get([]string{"flow.CreateAccount"}, 0, 0, 250, 1)
 
 		assert.NoError(t, err)
-		gw.Mock.AssertCalled(t, tests.GetEventsFunc, "flow.CreateAccount", uint64(0), uint64(1))
+		gw.Mock.AssertCalled(t, tests.GetEventsFunc, "flow.CreateAccount", uint64(0), uint64(0))
 	})
 
-	t.Run("Get Events Latest", func(t *testing.T) {
-		t.Parallel()
-
-		_, s, gw := setup()
-		_, err := s.Events.Get("flow.CreateAccount", "0", "latest")
-
-		assert.NoError(t, err)
-		gw.Mock.AssertCalled(t, tests.GetLatestBlockFunc)
-		gw.Mock.AssertCalled(t, tests.GetEventsFunc, "flow.CreateAccount", uint64(0), uint64(1))
-	})
-
-	t.Run("Fails to get events without name", func(t *testing.T) {
-		t.Parallel()
-
-		_, s, _ := setup()
-		inputs := [][]string{
-			{"", "0", "1"},
-			{"test", "-1", "1"},
-			{"test", "1", "-1"},
-			{"test", "10", "5"},
-		}
+	t.Run("Test create queries", func(t *testing.T) {
 
-		outputs := []string{
-			"cannot use empty string as event name",
-			"failed to parse start height of block range: -1",
-			"failed to parse end height of block range: -1",
-			"cannot have end height (5) of block range less that start height (10)",
```

Comment:
Is there a reason these failure tests were removed? I would like to leave them be.

There are no tests testing multiple workers which I think should be added since it is quite complex and it can break in the future by mistake.